core/web/resolver/query.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/graph-gophers/graphql-go"
 	"github.com/pkg/errors"
-
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
```

Comment:
I'm surprised the linter is not complaining about this:
```suggestion

	"github.com/smartcontractkit/chainlink-common/pkg/types"
```

Reply:
cool i added a new line to separate chainlink-common from the other 3rd party imports